addons/tmf_loadouts/loadouts/us_rangers_2010.hpp

```diff
@@ -0,0 +1,610 @@
+		"ItemCompass",
+		"ItemWatch",
+		"ACE_VectorDay",
+		"ItemGPS"
+	};
+};
+
+class co : sl {
+	displayName = "Platoon Leader";
+	headgear[] = {"H_MilCap_mcamo"};
+};
+
+class fac : co {
+	displayName = "Forward Air Controller";
+	backpack[] = {"B_RadioBag_01_mtp_F"};
+	backpackItems[] = { FAC_GEAR };
+	linkedItems[] = {
+		"ItemMap",
+		"ItemCompass",
+		"ItemWatch",
+		"CUP_SOFLAM",
+		"ItemGPS"
+	};
+};
+
+class ar : r {
+	displayName = "Automatic Rifleman";
+	primaryWeapon[] = {"hlc_lmg_mk46"};
+	attachment[] = {
+		"cup_acc_anpeq_15_flashlight_tan_l",
+		"cup_acc_anpeq_15"
+	};
+	sidearmWeapon[] = {"CUP_hgun_M9"};
+	backpack[] = {"B_AssaultPack_sgg"};
+	magazines[] = {
+		LIST_4("CUP_200Rnd_TE4_Red_Tracer_556x45_M249_Pouch"),
+		"HandGrenade",
+		"SmokeShell",
+		LIST_3("CUP_15Rnd_9x19_M9")
+	};
+};
+
+class aar : r {
+	displayName = "Assistant Automatic Rifleman";
+	backpack[] = {"B_Kitbag_sgg"};
+	backpackItems[] = {
+		LIST_3("CUP_200Rnd_TE4_Red_Tracer_556x45_M249_Pouch"),
+		"ACE_EntrenchingTool"
+	};
+	linkedItems[] += {"ACE_Yardage450"};
+};
+
+class rat : car {
+	displayName = "Rifleman (AT)";
+	secondaryWeapon[] = {"CUP_launch_M72A6_Loaded"};
+	backpackItems[] = {"CUP_launch_M72A6_Loaded"};
+};
+
+class dm : r {
+	displayName = "Designated Marksman";
+	primaryWeapon[] = {"CUP_srifle_m110_kac"};
+	scope[] = {"cup_optic_leupoldm31r"};
+	attachment[] = {"cup_acc_anpeq_2_grey"};
+	bipod[] = {"bipod_01_F_snd"};
+	magazines[] = {
+		LIST_9("CUP_20Rnd_762x51_B_M110"),
+		LIST_2("HandGrenade"),
+		LIST_2("SmokeShell")
+	};
+};
+
+class mmgg : ar {
+	displayName = "MMG Gunner";
+	primaryWeapon[] = {"CUP_lmg_Mk48"};
+	scope[] = {
+		"cup_optic_elcanm145",
+		"cup_optic_elcan_specterdr_coyote"
+	};
+	backPack[] = {"B_Kitbag_sgg"};
+	magazines[] = {
+		LIST_3("hlc_200Rnd_762x51_M_M60E4"),
+		"HandGrenade",
+		"SmokeShell",
+		LIST_3("CUP_15Rnd_9x19_M9")
+	};
+};
+
+class mmgac : aar {
+	displayName = "MMG Ammo Carrier";
+	backpackItems[] = {
+		LIST_2("hlc_200Rnd_762x51_M_M60E4")
+	};
+};
+
+class mmgtl : aar {
+	displayName = "MMG Team Leader";
+	items[] += {"ACE_MapTools"};
+	linkedItems[] = {
+		"ItemMap",
+		"ItemCompass",
+		"ItemWatch",
+		"ACE_Vector",
+		"ItemGPS"
+	};
+	backpackItems[] = {
+		LIST_2("CUP_100Rnd_TE4_LRT4_Red_Tracer_762x51_Belt_M")
+	};
+};
+
+class hmgg : car {
+	displayName = "HMG Gunner";
+	backPack[] = {"I_G_HMG_02_weapon_F"};
+};
+
+class hmgac : r {
+	displayName = "HMG Ammo Carrier";
+	backPack[] = {"I_C_HMG_02_support_F"};
+};
+
+class hmgtl : car {
+	displayName = "HMG Team Leader";
+	backPack[] = {"I_C_HMG_02_support_high_F"};
+	items[] += {"ACE_MapTools"};
+	linkedItems[] = {
+		"ItemMap",
+		"ItemCompass",
+		"ItemWatch",
+		"ACE_Vector",
+		"ItemGPS"
+	};
+};
+
+class matg : car {
+	displayName = "MAT Gunner";
+	backpack[] = {"B_Kitbag_sgg"};
+	secondaryWeapon[] = {"CUP_launch_MAAWS"};
+	secondaryAttachments[] = {"cup_optic_maaws_scope"};
+	magazines[] += {
+		LIST_2("MRAWS_HEAT_F"),
+		"MRAWS_HE_F"
+	};
```

Comment:
Might want to use CUP rounds instead vanillas

Reply:
Fix'd! I've replaced the vanilla rounds with the CUP rounds.